### DISCUSSION TITLE
problems with lancedb

### DISCUSSION THREAD

**Discussion by guangxiangdebizi** [microsoft/graphrag#880] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/deadad2b-46eb-4ea8-88f1-4df22356e587)
you see I have already got so much resources in output ,but why every times when i use the local method (such as :python -m graphrag.query  --root ./man --method local "How is Eren？" )to search the entities and relationships the engine only can search the lastest time resource in the output (in this picture which is the film named '20240809-145349')
![image](https://github.com/user-attachments/assets/dfb3ec24-3c82-43be-9ed1-32e20fb70378)
thanks for every answer
![image](https://github.com/user-attachments/assets/0bdc8a46-6f70-4e79-9fec-3246c6df6eda)
and this is my lancedb's location

**Comment by natoverse** [comment, marked as answer]
Every time you run `graphrag.index` it creates a new timestamp output folder. All of your index is recreated here, using the cache as much as possible. So you normally would want to search from the latest timestamp. If you use the `--resume <timestamp>` you can have the outputs write to an existing folder, but the purpose of that is to continue a run that stopped.

**Reply by guangxiangdebizi** [reply]
Thank you !
